### PR TITLE
Add Chromatic

### PR DESCRIPTION
**Pull request by gaetanmaisse**
**Issue**

Related to https://github.com/gravitee-io/gravitee-ui-components/issues/239

**Description**

### How does it work?

- Chromatic runs after each push on `master`.

- On PR it should be manually triggered (to avoid a gigantic bill) in CircleCI, there is a check on GitHub indicating or not if Chromatic can be started: 
  image

  On CircleCI you just need to click on:
  image
  And then approve, it will check if you have the correct right to do so (and automatically reduce your monthly salary of 0.03$ per snapshot. :trollface: )
  image

When Chromatic analysis is done checks are updated:
image

The first `Details` link redirects to the Chromatic App and allow you to accept or deny a change captured on a story. ⚠️ It will remain pending as long as there will be some changes not either accepted or denied.

The 2nd is a link to the deployed Storybook of the PR's branch.

### Snapshot rules

In order to avoid the maximum of false positive, the stories matching one of the following rules are ignored by Chromatic:
 - A story with a `loading` or `skeleton` property set to `true` 
 - A story defined with a non-empty `simulations` attribute (mainly used to fake async behaviour)
 - All the stories related to chart components (based on HighChart)
 - All the stories of `gv-relative-time` except `yearsAgo`
 - All the stories of `gv-spinner`

### Still to do

- Find a way to automatically open the dialog/modal in `gv-confirm`, `gv-dropdown` and `gv-popover` to get a snapshot of the interesting part of the component. It could be non-trivial and so will be done in another PR.

### CODE REVIEW

File: src/atoms/gv-relative-time.js [hunk 1]
```diff
@@ -145,7 +145,7 @@ export class GvRelativeTime extends LitElement {
   }
 
   render() {
-    return html`<span title="${this.title}">${until(this._format(this._datetime, getLanguage()))}</span>`;
+    return html`<span data-chromatic="ignore" title="${this.title}">${until(this._format(this._datetime, getLanguage()))}</span>`;
```
Comment (gaetanmaisse) [inline src/atoms/gv-relative-time.js:148]: poubelle

File: assets/css/codemirror/all.css [hunk 2]
```diff
@@ -19,7 +19,7 @@
 }
 
 .CodeMirror-scrollbar-filler, .CodeMirror-gutter-filler {
-  background-color: transparent; /* The little square between H and V scrollbars */
+  background-color: white; /* The little square between H and V scrollbars */
```
Comment (gaetanmaisse) [inline assets/css/codemirror/all.css:22]: I don't know why this changed 😮

**Comment by gaetanmaisse** [comment]
> UI Tests Pending — 97 changes must be accepted as baselines.

There are a lot of changes because the comparison base is a pretty old version of `master` as I init it for the demo a few weeks ago.

File: assets/css/codemirror/all.css (hunk 2 above)
Comment (gcusnieux) [reply assets/css/codemirror/all.css:22]: This file is generated https://github.com/gravitee-io/gravitee-ui-components/blob/master/tasks/generate-theme.js#L44-L51

Comment (gaetanmaisse) [reply assets/css/codemirror/all.css:22]: Is it ok to update it in this PR or this I revert it?

Comment (gcusnieux) [reply assets/css/codemirror/all.css:22]: The PR retrieve the last fix of codemirror in 5.59.4 https://codemirror.net/doc/releases.html. 
I think it's okay, but maybe we should be careful and put strict versions on some dependencies ?

**Review by gcusnieux — APPROVED** [review]

**Review by jgiovaresco — APPROVED** [review]

**Review by loriepisicchio — APPROVED** [review]

**Pull request merged**